src/cli.rs

```diff
@@ -656,21 +661,25 @@ impl Opts {
             (true, true) => panic!("partial and full are mutually exclusive"),
         };
 
-        if opts.names.is_empty() {
-            for (name, pin) in pins.pins.iter_mut() {
-                log::info!("Updating '{}' …", name);
```

Comment:
This line kinda got lost in the current code revision, wanna add it back in some form or another? Maybe the easiest way it to just put the line into the async block of the update task, however this needs trying out because it might look horrible in practice.

Reply:
Ideally the way it would be displayed would be using the same sort of idea as [nix-output-monitor](https://github.com/maralorn/nix-output-monitor) where you could see the in-progress updates. I'll add the log back for now but I do find it personally rather annoying. 

Maybe a quick fix would be to only log it if `update_one` takes a significantly long time to update (like it would for nixpkgs)?